src/Components/Components/src/EventCallbackWorkItem.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Components
+{
+    /// <summary>
+    /// Wraps a callback delegate associated with an event.
+    /// </summary>
+    public struct EventCallbackWorkItem
```

Comment:
Suggestions welcome. We want to find an unappealing name for this time so you're likely to use `EventCallback` which is what we want users to find. We could put this in another namespace, but none of the ones we have already are really appropriate. Ultimately we might want to more component lifecycle interfaces elsewhere, and this type would follow.

Reply:
I think this name sounds unappealing enough as-is.